### PR TITLE
Fix/environment variables

### PR DESCRIPTION
**Pull request by mzyau**
1) Set up environment variables in appropriate files using dotenv library
2) Clean up some tests
3) Update readme for latest process

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,4 @@ jwtRS256.key
 jwtRS256.key.pub
 keygen.sh
 database.json
-.env*
+.env*
```
Comment (dadiorchen) [inline .gitignore:9]: Haha, interesting, what is that? a \n ? it would lead to problem?

File: package.json [hunk 2]
```diff
@@ -10,11 +10,11 @@
     "eslint": "eslint --report-unused-disable-directives .",
     "eslint:fix": "npm run eslint -- --fix",
     "test": "npm run test-unit; npm run test-integration;npm run test-repository",
-    "test-unit": "NODE_ENV=test mocha -r dotenv/config dotenv_config_path=.env.test --exit ./server/models/**/*.spec.js ./server/routes/**/*.spec.js",
-    "test-repository": "NODE_ENV=test mocha -r dotenv/config --exit --require co-mocha  ./server/repositories/**/*.spec.js",
+    "test-unit": "NODE_ENV=test mocha -r dotenv/config dotenv_config_path=.env.test --exit --require co-mocha ./server/models/**/*.spec.js ./server/routes/**/*.spec.js",
```
Comment (dadiorchen) [inline package.json:13]: I still don't get the meaning of this? what's the `--require` for?

**Review by dadiorchen — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (mzyau) [reply .gitignore:9]: No, sorry I just removed it by accident in a commit

File: package.json (hunk 2 above)
Comment (mzyau) [reply package.json:13]: I think it works fine without it, I was just playing around with the scripts and kept it in for consistency with some others.

**Pull request merged**